### ISSUE TITLE
No results displayed

### ISSUE THREAD

**Issue by atuttle** [seancoyne/testbox-runner#5]
my bundle resembles:

```
describe("", function(){
  describe("", function(){
    it("", function(){
        //...
    });
  });
});
```

My tests pass in the web runner:

![image](https://cloud.githubusercontent.com/assets/46990/10701799/9d19c224-7993-11e5-96c3-f2223b8ad29a.png)

And when I run the URL reported by testbox-runner:

![image](https://cloud.githubusercontent.com/assets/46990/10701815/be1677a6-7993-11e5-9906-905efeae5fae.png)

... I get what seems like a reasonable json response:

![image](https://cloud.githubusercontent.com/assets/46990/10701840/d6bc71d4-7993-11e5-82d6-8faf72eaf80d.png)

But there's no response displayed in the CLI (as you can see from that next-to-last screen shot)

I tried upgrading from Testbox 2.0 to 2.2, but that didn't fix it.

**Comment by seancoyne** [comment]
what version of node are you running?

**Comment by atuttle** [comment]
4.1.1

**Comment by seancoyne** [comment]
can you copy the JSON you get back when you run the URL in a browser into a text file and share that?  I want to use it to test the output.

**Comment by atuttle** [comment]
https://gist.github.com/atuttle/98de4590e60397ea2911

**Comment by seancoyne** [comment]
OK, I found the issue.  It is your nested suites (the describe() blocks).  The app doesn't expect that.  I'll prep a fix.

**Comment by atuttle** [comment]
:+1:

**Comment by seancoyne** [comment]
ok should be available on npm as version 0.2.0

**Comment by atuttle** [comment]
Confirmed working. Thanks!
